inst/extdata/energy/mappings/UCD_techs.csv

```diff
@@ -0,0 +1 @@
Freight,Truck,Truck (6-14t),NG,Natural Gas,trn_road and rail,gas,trn_freight_road,Truck (6-14t),NG,delivered gas
Freight,Truck,Truck (>14t),Liquids,Liquids,trn_road and rail,refined liquids,trn_freight_road,Truck (>14t),Liquids,refined liquids enduse
Freight,Truck,Truck (>14t),NG,Natural Gas,trn_road and rail,gas,trn_freight_road,Truck (>14t),NG,delivered gas
Freight,Truck,Truck (0-3.5t),Liquids,Liquids,trn_road and rail,refined liquids,trn_freight_road,Truck (0-3.5t),Liquids,refined liquids enduse
Freight,Truck,Truck (0-3.5t),NG,Natural Gas,trn_road and rail,gas,trn_freight_road,Truck (0-3.5t),NG,delivered gas
Freight,Truck,Truck (3.5-16t),Liquids,Liquids,trn_road and rail,refined liquids,trn_freight_road,Truck (3.5-16t),Liquids,refined liquids enduse
Freight,Truck,Truck (3.5-16t),NG,Natural Gas,trn_road and rail,gas,trn_freight_road,Truck (3.5-16t),NG,delivered gas
Freight,Truck,Truck (16-32t),Liquids,Liquids,trn_road and rail,refined liquids,trn_freight_road,Truck (16-32t),Liquids,refined liquids enduse
Freight,Truck,Truck (16-32t),NG,Natural Gas,trn_road and rail,gas,trn_freight_road,Truck (16-32t),NG,delivered gas
Freight,Truck,Truck (>32t),Liquids,Liquids,trn_road and rail,refined liquids,trn_freight_road,Truck (>32t),Liquids,refined liquids enduse
Freight,Truck,Truck (>32t),NG,Natural Gas,trn_road and rail,gas,trn_freight_road,Truck (>32t),NG,delivered gas
Passenger,LDV_2W,Scooter,Liquids,Liquids,trn_road and rail,refined liquids,trn_pass_road_LDV_2W,Scooter,Liquids,refined liquids enduse
Passenger,LDV_2W,Scooter,BEV,Electricity,trn_road and rail,electricity,trn_pass_road_LDV_2W,Scooter,BEV,elect_td_trn
Freight,Truck,Truck,Liquids,Liquids,trn_road and rail,refined liquids,trn_freight_road,Truck,Liquids,refined liquids enduse
Freight,Truck,Truck,NG,Natural Gas,trn_road and rail,gas,trn_freight_road,Truck,NG,delivered gas
Freight,Truck,Truck (0-2.7t),Liquids,Liquids,trn_road and rail,refined liquids,trn_freight_road,Truck (0-2.7t),Liquids,refined liquids enduse
Freight,Truck,Truck (0-2.7t),NG,Natural Gas,trn_road and rail,gas,trn_freight_road,Truck (0-2.7t),NG,delivered gas
Freight,Truck,Truck (2.7-4.5t),Liquids,Liquids,trn_road and rail,refined liquids,trn_freight_road,Truck (2.7-4.5t),Liquids,refined liquids enduse
Freight,Truck,Truck (2.7-4.5t),NG,Natural Gas,trn_road and rail,gas,trn_freight_road,Truck (2.7-4.5t),NG,delivered gas
Freight,Truck,Truck (4.5-12t),Liquids,Liquids,trn_road and rail,refined liquids,trn_freight_road,Truck (4.5-12t),Liquids,refined liquids enduse
Freight,Truck,Truck (4.5-12t),NG,Natural Gas,trn_road and rail,gas,trn_freight_road,Truck (4.5-12t),NG,delivered gas
Freight,Truck,Truck (>12t),Liquids,Liquids,trn_road and rail,refined liquids,trn_freight_road,Truck (>12t),Liquids,refined liquids enduse
Freight,Truck,Truck (>12t),NG,Natural Gas,trn_road and rail,gas,trn_freight_road,Truck (>12t),NG,delivered gas
```

Comment:
Here and in all files, please remove ugly trailing Excel commas from header lines.